pkgs/os-specific/linux/wireguard/0.0.20161209.nix

```diff
@@ -0,0 +1,9 @@
+{ stdenv, fetchurl, libmnl, iproute, kernel ? null }:
+
+# 0.0.20161216 introduced a non-backwards protocol change.
+# this version is kept around to have a version compatible with nixos stable.
```

Comment:
No. This is not okay. Do not keep this old version around. Stability with the wireguard snapshots has NEVER been guaranteed and there have been several warnings about it. It will be broken away. Absolutely do not merge this old version into Nix.